.eleventy.js

```diff
@@ -0,0 +1,17 @@
+const yaml = require("js-yaml");
+
+module.exports = function(eleventyConfig) {
+    eleventyConfig.addDataExtension("yaml", (contents, filePath) => {
+        console.log("Trying to process ", filePath);
+        return yaml.load(contents);
+    });
+
+
+    //eleventyConfig.setServerPassthroughCopyBehavior("passthrough");
+
+    // Copy any .svg or midi file to `_site`, via Glob pattern
+    // Keeps the same directory structure
+    eleventyConfig.addPassthroughCopy({"songs/*.svg": "songs"});
+    eleventyConfig.addPassthroughCopy({"songs/*.midi": "songs"});
```

Comment:
All svgs/midis are stored in the same folder. It is all based on naming convention. This should be changed in the future

Reply:
yes